tests/_support/Page/Framework.php

```diff
@@ -1414,4 +1414,21 @@ public function iCreateACustomField()
 
         $I->see('test_additionalfield', Locator::combine('//table/tbody/tr', -1));
     }
+
+    /**
+     * @Then /^I delete the custom field$/
+     * @Then /^I delete a custom field$/
+     */
+    public function iDeleteACustomField()
+    {
+        $I = $this->I;
+
+        $I->amOnPage(self::$additionalFieldPath);
+        $I->see('test_additionalfield');
+
+        $I->click('Delete', Locator::combine('//table/tbody/tr', -1));
+        $I->wait(5);
```

Comment:
Is the wait necessary here?  It is advised to avoid timed waits if there are other options (to allow the test to finish faster).